src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/ServiceBusTriggerBinding.cs

```diff
@@ -141,6 +141,7 @@ public Task<IListener> CreateListenerAsync(ListenerFactoryContext context)
             contract.Add("To", typeof(string));
             contract.Add("Label", typeof(string));
             contract.Add("CorrelationId", typeof(string));
+            contract.Add("Properties", typeof(IDictionary<string, object>));
```

Comment:
Doing the same thing for SB - this will allow people to access custom message properties.